test/Api.Tests/appsettings.example.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "AppSettings": {
+    "SiteTitle": "ASP.NET CORE HIP APP",
+    "CacheDbResults": true
+  },
+  "DefaultAdminUsername": "Administrator@test.com",
```

Comment:
Looks like you don't use these variables. Please consider which settings/values you really need.